package.json

```diff
@@ -82,7 +83,7 @@
 		"@nextcloud/babel-config": "^1.0.0",
 		"@nextcloud/browserslist-config": "^2.3.0",
 		"@nextcloud/cypress": "^1.0.0-beta.2",
-		"@nextcloud/eslint-config": "^8.0.0",
+		"@nextcloud/eslint-config": "^8.2.1",
```

Comment:
typescript 5 support if you need btw  https://github.com/nextcloud/eslint-config/releases/tag/v8.3.0-beta.0